groundstation/static/js/components/Help.jsx

```diff
@@ -5,212 +5,381 @@ import Button from '@material-ui/core/Button';
+              <TextField
+                id="outlined-basic"
+                label="Delta Time"
+                variant="outlined"
+                type="number"
+              />
+            </form>
+            The delta time is an offset from the scheduled time in seconds. This
+            means the command will execute x seconds after the scheduled time.
+            You can also delete any command in the flight schedule using the{' '}
+            <DeleteIcon style={{ color: '#4bacb8' }} /> for that command.
+          </Typography>
+        </Paper>
+        <Paper className="Housekeeping" style={{ marginBottom: '20px' }}>
+          <Typography variant="h5" style={{ padding: '10px' }}>
+            Housekeeping
+          </Typography>
+          <Typography
+            variant="body1"
+            style={{
+              paddingLeft: '20px',
+              paddingRight: '20px',
+              paddingBottom: '10px'
+            }}
+          >
+            Use Housekeeping to view the status and health of the satellite.
+            Each housekeeping log contains information about the satellite, such
+            as the watchdog timers, as well as voltages. A housekeeping file
+            with a green tag means that the housekeeping log is healthy and
+            detected no issues,
+            <Paper
+              style={{
+                marginLeft: '20px',
+                borderLeft: 'solid 8px #479b4e',
+                marginBottom: '10px',
+                marginTop: '10px',
+                marginRight: '20px'
+              }}
+            >
+              <div style={{ fontWeight: 'bold', marginLeft: '4px' }}>
+                Healthy Housekeeping Log
+              </div>
+            </Paper>
+            whereas a red tag signifies that there may be an issue with the
+            satellite.
+            <Paper
+              style={{
+                marginLeft: '20px',
+                borderLeft: 'solid 8px #721c24',
+                marginBottom: '10px',
+                marginTop: '10px',
+                marginRight: '20px'
+              }}
+            >
+              <div style={{ fontWeight: 'bold', marginLeft: '4px' }}>
+                Critical Housekeeping Log
+              </div>
+            </Paper>
+            You can also filter housekeeping logs by timestamp using the filter
+            options.
+          </Typography>
+        </Paper>
+        <Paper className="Live Commands" style={{ marginBottom: '20px' }}>
+          <Typography variant="h5" style={{ padding: '10px' }}>
+            Live Commands
+          </Typography>
+          <Typography
+            variant="body1"
+            style={{
+              paddingLeft: '20px',
+              paddingRight: '20px',
+              paddingBottom: '10px'
+            }}
+          >
+            Send telecommands to the satellite via the live commands page.
+            Commands are formatted as "prefix.command_name(arg1 arg2 ...) "
```

Comment:
```suggestion
            Commands are formatted as "prefix.command_name(arg1,arg2,arg3,...) "
```
The command format has changed where args are now separated by a comma